compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/scene/UIViewComposeSceneLayer.uikit.kt

```diff
@@ -33,24 +33,53 @@ import androidx.compose.ui.window.ProvideContainerCompositionLocals
 import androidx.compose.ui.window.RenderingUIView
 import kotlin.coroutines.CoroutineContext
 import kotlinx.cinterop.CValue
+import kotlinx.cinterop.readValue
+import platform.CoreGraphics.CGPoint
+import platform.CoreGraphics.CGRectZero
 import platform.CoreGraphics.CGSize
+import platform.UIKit.NSLayoutConstraint
+import platform.UIKit.UIColor
+import platform.UIKit.UIEvent
 import platform.UIKit.UIView
 import platform.UIKit.UIViewControllerTransitionCoordinatorProtocol
 
 internal class UIViewComposeSceneLayer(
     private val composeContainer: ComposeContainer,
-    density: Density,
-    layoutDirection: LayoutDirection,
+    private val initDensity: Density,
+    private val initLayoutDirection: LayoutDirection,
     configuration: ComposeUIViewControllerConfiguration,
     focusStack: FocusStack<UIView>?,
     windowInfo: WindowInfo,
     compositionContext: CompositionContext,
     compositionLocalContext: CompositionLocalContext?,
 ) : ComposeSceneLayer {
 
+    override var focusable: Boolean = focusStack != null
+    private var onOutsidePointerEvent: ((dismissRequest: Boolean) -> Unit)? = null
+    private val rootView = composeContainer.view.window ?: composeContainer.view
```

Comment:
If composeContainer is detached from window, we're using its view, which is safety measure, but not the way how compose sub-layer is expected to work.
If we have control over `ComposeContainer`, we can detect when it become attached (and detached) in window and use this fact to attach(detach) other scene layers.

Reply:
In fact rootView is always presented now and it works like expected. Also I we will change this behaviour to use multiple UIWindow - this place we be cnahged again.
So, maybe let's stay as is for now?